config/config.default.php

```diff
@@ -1,53 +1,66 @@
 <?php
 /**
- * Default configuration for Xhgui
+ * Default configuration for XHGui.
+ *
+ * To change these, create a called `config.php` file in the same directory,
+ * and return an array from there with your overriding settings.
  */
 
-$mongoUri = getenv('XHGUI_MONGO_URI') ?: '127.0.0.1:27017';
-$mongoUri = str_replace('mongodb://', '', $mongoUri);
-$mongoDb = getenv('XHGUI_MONGO_DB') ?: 'xhprof';
-
 return array(
-    'debug' => false,
-    'mode' => 'development',
+    'debug'     => false,
+    'mode'      => 'development',
```

Comment:
please revert all formatting changes that do not change code.

Reply:
or if you insist formatting changes, we should do that in separate PR with a tool that controls it (php-cs-fixer for example).